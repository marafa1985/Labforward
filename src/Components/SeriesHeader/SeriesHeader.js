import React,{useState} from 'react';
import PropTypes from 'prop-types';

const SeriesHeader = (props) => {
    const [seriesValue, setseriesValue] = useState('')
    return (
        <div>
            <input className="input-text" type="text" placeholder="Series name"
                value={seriesValue} onChange={e => setseriesValue(e.target.value)} />
            <button className="btn" onClick={() => props.addNewSeries(seriesValue)}>New Series</button>
        </div>
    )
}

SeriesHeader.propTypes = {
    addNewSeries: PropTypes.func
}
export default SeriesHeader
