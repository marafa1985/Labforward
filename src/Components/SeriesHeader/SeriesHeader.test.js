import React from 'react';
import { create } from "react-test-renderer";
import SeriesHeader from './SeriesHeader';

it('renders without crashing', () => {
    const seriesHeader = create(<SeriesHeader />);
  expect(seriesHeader.toJSON()).toMatchSnapshot()
});
