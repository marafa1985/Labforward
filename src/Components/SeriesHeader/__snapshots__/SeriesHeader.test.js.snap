// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div>
  <input
    className="input-text"
    onChange={[Function]}
    placeholder="Series name"
    type="text"
    value=""
  />
  <button
    className="btn"
    onClick={[Function]}
  >
    New Series
  </button>
</div>
`;
